test/e2e/scheduler_predicates.go

```diff
@@ -110,6 +110,22 @@ func verifyResult(c *client.Client, podName string, ns string) {
 			"reason":                   "FailedScheduling",
 		}.AsSelector())
 	expectNoError(err)
+	// If we failed to find event with a capitalized first letter of reason
+	// try looking for one starting with a small one for backward compatibility.
+	// If we don't do it we end up in #158061.
+	// TODO: remove this block when we don't care about supporting v1.0 too much.
```

Comment:
Can't we just use github tags for it? Such solution will give way more context that separate issue.